### DISCUSSION TITLE
Tweaks for tv shows

### DISCUSSION THREAD

**Discussion by overtoneblue** [itsToggle/plex_debrid#127] [Ideas]
Hi, I think it would be a nice option to separate the maximum release size for movies vs tv shows, reason being is some tv shows with large seasons will have large overall file sizes if an entire season is found in one source. One other issue I found is that sometimes when multi seasons packs are found the script will sacrifice or discount the desired resolutions in order to find multi season packs. For example the tv show archer, there are some multi season packs that have multiple seasons, however the resolution is in 720 or even 360p, resulting in a lower than desired resolution being selected, this often happens for tv shows that have had a long running time, you can try replicating with the animated tv show "archer" however our indexers may differ.

**Comment by itsToggle** [comment]
Hi! I thought I fixed this already, here is what the script is currently doing if more than 3 seasons are to be downloaded:

- Scrape for "some.show" (no season/episode specified). This returns almost all releases related to this show.  
- Check if there are multi-season packs available and note the highest resolution 
- Check if there is a season pack available for every season and note the highest resolution
- If not all seasons can be downloaded as single season packs, or the highest resolution for the single season packs is equal to or lower than the highest resolution of the best multi season pack, download the multi season pack.

This should usually work, but I think I just found an error in this process. i will have a look at it again. Would you mind telling me the quality you were aiming for and if there was a multi-season pack of higher quality available? (e.g. you wanted to download 1080p, there was a 2160p multi-season pack, the script chose to download a 720p multi season pack, eventhough the single season packs could have been downloaded in 1080p)
